portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+public final class Comment {
+  private final long id;
+  private final String name;
+  private final String content;
+  private final long timestamp;
+
+  public Comment(long id, String name, String content, long timestamp) {
```

Comment:
For public methods, try to give some comments for it. For example, what are these parameters.